mteb/abstasks/AbsTaskClusteringFast.py

```diff
@@ -91,11 +91,12 @@ class AbsTaskClusteringFast(AbsTask):
         labels: list[str] | list[list[str]]
     """
 
-    max_documents_to_embed = 16_384
-    max_documents_per_cluster = 2048
+    max_fraction_of_documents_to_embed = 0.04
+    max_documents_per_cluster = 16_384
     n_clusters = 10
     k_mean_batch_size = 512
     max_depth = None
+    use_dataset_as_is = False
```

Comment:
```suggestion
```

Reply:
This param is used in `RedditFastClusteringS2S` and `StackExchangeClusteringFast`, where the data is subsampled during `dataset_transform` to 32768 n_samples.

Reply:
I think if both of the above is None, that would be the same as having this true?